src/UserInfoHandler.ts

```diff
@@ -0,0 +1,42 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Response } from "./utils/Response";
+import { ClaimedIdRequestProcessor } from "./services/ClaimedIdRequestProcessor";
+import { ResourcesEnum } from "./models/enums/ResourcesEnum";
+import { AppError } from "./utils/AppError";
+import { HttpCodesEnum } from "./utils/HttpCodesEnum";
+import { UserInfoRequestProcessor } from "./services/UserInfoRequestProcessor";
+
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : "CIC-CRI";
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : "DEBUG";
+
+const logger = new Logger({
+	logLevel: POWERTOOLS_LOG_LEVEL,
+	serviceName: "UserInfoHandler",
+});
+
+const metrics = new Metrics({ namespace: POWERTOOLS_METRICS_NAMESPACE });
+
+export const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+	switch (event.resource) {
+
+		case ResourcesEnum.USERINFO:
+			if (event.httpMethod === "POST") {
```

Comment:
Think we can remove this as per @CoralSmith's comment here: https://github.com/alphagov/di-ipv-cri-cic-api/pull/17#discussion_r1112064606

Reply:
Will have a look and remove it. Thanks.

Reply:
Not changing this as we are still in discussion.